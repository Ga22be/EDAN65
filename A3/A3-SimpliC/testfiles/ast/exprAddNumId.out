Program
  List
    Function
      IdDecl ID="add"
      List
      List
        Declaration
          IdDecl ID="a"
        Declaration
          IdDecl ID="b"
        Assignment
          IdUse ID="b"
          Numeral NUMERAL="7"
        Assignment
          IdUse ID="a"
          Add
            Numeral NUMERAL="9"
            IdUse ID="b"
