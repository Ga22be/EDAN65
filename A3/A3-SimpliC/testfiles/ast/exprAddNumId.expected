Program
  List
    Function
      IdDecl ID="add"
      List
      List
        Declaration
          IdDecl ID="a"
        Declaration
          IdDecl ID="b"
        Assignment
          IdDecl ID="b"
          Numeral NUMERAL="7"
        Assignment
          IdDecl ID="a"
          Add
            Numeral NUMERAL="9"
            IdUse ID="b"
